### DISCUSSION TITLE
graphql/envelop/plugins/graphql-armor-max-depth

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/envelop#1592] [Docs Discussions]
# graphql/envelop/plugins/graphql-armor-max-depth

GraphQL Envelop Documentation

https://the-guild.dev/graphql/envelop/plugins/graphql-armor-max-depth

**Comment by lexi2042** [comment]
the `maxDirectivesPlugin` entry from the explore page links to the `maxDepthPlugin` page.
